packages/client/src/hoc/withChecker.tsx

```diff
@@ -1,11 +1,11 @@
+import getChecker from "@vergunningcheck/imtr_client";
```

Comment:
Why is `getChecker` not imported as `{ getChecker }` ?

Reply:
Good point, I guess I can make the default export a named one.

While changing this ... the test didn't fail! That concerned me a bit.
I'm refactoring now and realize I should have made `packages/imtr_client/src/index.js` a typescript file as well. I'll add it to my list because it takes a bit of time to do that.